src/components/select/filterable-select/filterable-select.component.js

```diff
@@ -596,9 +596,6 @@ FilterableSelect.propTypes = {
   listMaxHeight: PropTypes.number,
   /** Placement of the select list in relation to the input element */
   listPlacement: PropTypes.oneOf([
-    "auto",
-    "auto-start",
-    "auto-end",
     "top",
     "top-start",
     "top-end",
```

Comment:
so if floatingUi no longer has these "auto" placement options and we take them in as a prop, I guess this becomes a breaking change, as anyone using one of them will now get a TS/PropTypes error and it will do nothing anyway.